everviz/plugins/objectives_plot/objectives_plot.py

```diff
@@ -181,10 +181,11 @@ def set_button_enabled_state(filter_realizations):
                 Input(self.radio_id_mode, "value"),
                 Input(self.realization_filter_check_id, "value"),
                 Input(self.realization_filter_input_id, "value"),
+                Input(self.graph_id, "hoverData"),
```

Comment:
I would replace `hoverData` this with `clickData`

Using hoverData causes some strange flicker effect and hides the values of the points user is hovering over